### PR TITLE
Fixed RTL support and overflow text in cells.

### PR DESCRIPTION
**Pull request by kegor**
Replicated changes from https://github.com/groupdocs-viewer/GroupDocs.Viewer-for-.NET-MVC/pull/60.

### CODE REVIEW

File: src/Products/Viewer/Controllers/ViewerApiController.cs [hunk 1]
```diff
@@ -41,6 +41,9 @@ public class ViewerApiController : ApiController
         /// </summary>
         public ViewerApiController()
         {
+            // Check if filesDirectory is relative or absolute path           
+            globalConfiguration = new Common.Config.GlobalConfiguration();
```
Comment (egorovpavel) [inline src/Products/Viewer/Controllers/ViewerApiController.cs:45]: Codacy Issue found: [Remove this assignment of 'globalConfiguration' or initialize it statically.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Viewer-for-.NET-WebForms/pullRequest?prid=5485688)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Viewer-for-.NET-WebForms/pullRequest?prid=5485688&bid=17975915)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-viewer/GroupDocs.Viewer-for-.NET-WebForms/pullRequest?prid=5485687&bid=17975914)

File: src/Products/Viewer/Controllers/ViewerApiController.cs (hunk 1 above)
Comment (bobkovalex) [inline src/Products/Viewer/Controllers/ViewerApiController.cs:45]: Codacy Issue found: [Remove this assignment of 'globalConfiguration' or initialize it statically.](https://app.codacy.com/gh/groupdocs-viewer/GroupDocs.Viewer-for-.NET-WebForms/pullRequest?prid=5485687)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 8
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-viewer/GroupDocs.Viewer-for-.NET-WebForms/pullRequest?prid=5485687&bid=17975914)

File: src/Products/Common/Config/GlobalConfiguration.cs [hunk 2]
```diff
@@ -7,10 +7,10 @@ namespace GroupDocs.Viewer.WebForms.Products.Common.Config
     /// </summary>
     public class GlobalConfiguration
     {
-        public ServerConfiguration Server { get; set; }
-        public ApplicationConfiguration Application { get; set; }
-        public CommonConfiguration Common { get; set; }
-        public ViewerConfiguration Viewer { get; set; }
+        public ServerConfiguration Server;
+        public ApplicationConfiguration Application;
+        public CommonConfiguration Common;
```
Comment (bobkovalex) [inline src/Products/Common/Config/GlobalConfiguration.cs:12]: Codacy Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/groupdocs-viewer/GroupDocs.Viewer-for-.NET-WebForms/pullRequest?prid=5485687)

File: src/Products/Common/Config/GlobalConfiguration.cs [hunk 3]
```diff
@@ -7,10 +7,10 @@ namespace GroupDocs.Viewer.WebForms.Products.Common.Config
     /// </summary>
     public class GlobalConfiguration
     {
-        public ServerConfiguration Server { get; set; }
-        public ApplicationConfiguration Application { get; set; }
-        public CommonConfiguration Common { get; set; }
-        public ViewerConfiguration Viewer { get; set; }
+        public ServerConfiguration Server;
+        public ApplicationConfiguration Application;
```
Comment (bobkovalex) [inline src/Products/Common/Config/GlobalConfiguration.cs:11]: Codacy Issue found: [Make 'Application' private.](https://app.codacy.com/gh/groupdocs-viewer/GroupDocs.Viewer-for-.NET-WebForms/pullRequest?prid=5485687)

File: src/Products/Common/Config/GlobalConfiguration.cs (hunk 2 above)
Comment (bobkovalex) [inline src/Products/Common/Config/GlobalConfiguration.cs:12]: Codacy Issue found: [Make 'Common' private.](https://app.codacy.com/gh/groupdocs-viewer/GroupDocs.Viewer-for-.NET-WebForms/pullRequest?prid=5485687)

File: src/Products/Common/Config/GlobalConfiguration.cs [hunk 4]
```diff
@@ -7,10 +7,10 @@ namespace GroupDocs.Viewer.WebForms.Products.Common.Config
     /// </summary>
     public class GlobalConfiguration
     {
-        public ServerConfiguration Server { get; set; }
-        public ApplicationConfiguration Application { get; set; }
-        public CommonConfiguration Common { get; set; }
-        public ViewerConfiguration Viewer { get; set; }
+        public ServerConfiguration Server;
```
Comment (bobkovalex) [inline src/Products/Common/Config/GlobalConfiguration.cs:10]: Codacy Issue found: [Make 'Server' private.](https://app.codacy.com/gh/groupdocs-viewer/GroupDocs.Viewer-for-.NET-WebForms/pullRequest?prid=5485687)

File: src/Products/Common/Config/GlobalConfiguration.cs (hunk 3 above)
Comment (bobkovalex) [inline src/Products/Common/Config/GlobalConfiguration.cs:11]: Codacy Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/groupdocs-viewer/GroupDocs.Viewer-for-.NET-WebForms/pullRequest?prid=5485687)

File: src/Products/Common/Config/GlobalConfiguration.cs [hunk 5]
```diff
@@ -7,10 +7,10 @@ namespace GroupDocs.Viewer.WebForms.Products.Common.Config
...
+        public ApplicationConfiguration Application;
+        public CommonConfiguration Common;
+        public ViewerConfiguration Viewer;
```
Comment (bobkovalex) [inline src/Products/Common/Config/GlobalConfiguration.cs:13]: Codacy Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/groupdocs-viewer/GroupDocs.Viewer-for-.NET-WebForms/pullRequest?prid=5485687)

Comment (bobkovalex) [inline src/Products/Common/Config/GlobalConfiguration.cs:13]: Codacy Issue found: [Make 'Viewer' private.](https://app.codacy.com/gh/groupdocs-viewer/GroupDocs.Viewer-for-.NET-WebForms/pullRequest?prid=5485687)

File: src/Products/Common/Config/GlobalConfiguration.cs (hunk 4 above)
Comment (bobkovalex) [inline src/Products/Common/Config/GlobalConfiguration.cs:10]: Codacy Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/groupdocs-viewer/GroupDocs.Viewer-for-.NET-WebForms/pullRequest?prid=5485687)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 8
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Viewer-for-.NET-WebForms/pullRequest?prid=5485688&bid=17975915)

File: src/Products/Common/Config/GlobalConfiguration.cs (hunk 4 above)
Comment (egorovpavel) [inline src/Products/Common/Config/GlobalConfiguration.cs:10]: Codacy Issue found: [Make 'Server' private.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Viewer-for-.NET-WebForms/pullRequest?prid=5485688)

File: src/Products/Common/Config/GlobalConfiguration.cs (hunk 2 above)
Comment (egorovpavel) [inline src/Products/Common/Config/GlobalConfiguration.cs:12]: Codacy Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Viewer-for-.NET-WebForms/pullRequest?prid=5485688)

File: src/Products/Common/Config/GlobalConfiguration.cs (hunk 3 above)
Comment (egorovpavel) [inline src/Products/Common/Config/GlobalConfiguration.cs:11]: Codacy Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Viewer-for-.NET-WebForms/pullRequest?prid=5485688)

File: src/Products/Common/Config/GlobalConfiguration.cs (hunk 5 above)
Comment (egorovpavel) [inline src/Products/Common/Config/GlobalConfiguration.cs:13]: Codacy Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Viewer-for-.NET-WebForms/pullRequest?prid=5485688)

File: src/Products/Common/Config/GlobalConfiguration.cs (hunk 2 above)
Comment (egorovpavel) [inline src/Products/Common/Config/GlobalConfiguration.cs:12]: Codacy Issue found: [Make 'Common' private.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Viewer-for-.NET-WebForms/pullRequest?prid=5485688)

File: src/Products/Common/Config/GlobalConfiguration.cs (hunk 3 above)
Comment (egorovpavel) [inline src/Products/Common/Config/GlobalConfiguration.cs:11]: Codacy Issue found: [Make 'Application' private.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Viewer-for-.NET-WebForms/pullRequest?prid=5485688)

File: src/Products/Common/Config/GlobalConfiguration.cs (hunk 5 above)
Comment (egorovpavel) [inline src/Products/Common/Config/GlobalConfiguration.cs:13]: Codacy Issue found: [Make 'Viewer' private.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Viewer-for-.NET-WebForms/pullRequest?prid=5485688)

File: src/Products/Common/Config/GlobalConfiguration.cs (hunk 4 above)
Comment (egorovpavel) [inline src/Products/Common/Config/GlobalConfiguration.cs:10]: Codacy Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Viewer-for-.NET-WebForms/pullRequest?prid=5485688)

**Pull request merged**